oauth_tools/external_idp.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import abc
```

Comment:
So the `dex.py` in `tests/integration` will also be gone, correct?

Reply:
correct, removed